modules/ROOT/pages/configure-message-flows.adoc

```diff
@@ -0,0 +1,20 @@
+= Create a Receive Message Flow from Your Partner
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+When receiving a B2B message from your trading partner, it may include more than one URL. For example, a receiving URL, and a URL for receiving asynchronous MDNs (Message Disposition Notifications). If this is the case, you only need to configure the Receiving URL. If a separate URL is required for returning MDNs, it is sent in the AS2 transmission and Partner Manager reads it from there.
```

Comment:
Before launching into specifics, maybe say why or in what context (or what part of the process as illustrated in a diagram?) this is for. Do you receive a message flow, or a flow of messages? I thought message flows described the channel...?

Reply:
"it is sent" what is sent? the separate URL for returning MDNs? Do I have to do any configuring for that to happen or is it automatic?

Reply:
Anypoint Partner Manager unless leaving off Anypoint is approved.

Reply:
The PM says the product name has Anypoint in front of it. So stay it must.